apps/cloudfront-functions/src/viewer-request-handler.ts

```diff
@@ -13,8 +13,12 @@ const handler = (
     const { request } = event;
     const uri = request.uri;
 
-    // Add the .html extension if missing
-    if (!uri.endsWith('/') && !/\.[a-zA-Z]+$/.test(uri)) {
+    // Add the .html extension if missing and the uri does not end with .woff2
+    if (
+      !uri.endsWith('/') &&
+      !uri.endsWith('.woff2') &&
+      !/\.[a-zA-Z]+$/.test(uri)
```

Comment:
```suggestion
      !/\.[0-9a-zA-Z]+$/.test(uri)
```

I would just make this more generic and prevent the program from adding '.html' to any url ending in any extension which contains letters or numbers (hence including .woff2)